build/crds/devices/devices_v1beta1_device.yaml

```diff
@@ -600,6 +600,19 @@ spec:
                             data to database, please ensure that the mapper can access
                             the destination address.
                           properties:
+                            TDEngine:
+                              properties:
+                                configData:
+                                  description: ConfigData of tdengine database
+                                  properties:
+                                    DB:
+                                      description: addr of tdEngine database
+                                      type: string
+                                    dsn:
+                                      description: addr of tdEngine database
+                                      type: string
```

Comment:
1. the description of DB and dsn is the same, need to distinguish
2. change "configdata" to "tdengineClientConfig" may be better